test/Integration.t.sol

```diff
@@ -0,0 +1,311 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+pragma solidity ^0.8.13;
+
+import "forge-std/Test.sol";
+
+import { ERC20 } from "openzeppelin-contracts/token/ERC20/ERC20.sol";
+
+import { ReserveConfiguration } from "aave-v3-core/contracts/protocol/libraries/configuration/ReserveConfiguration.sol";
+import { DataTypes }            from "aave-v3-core/contracts/protocol/libraries/types/DataTypes.sol";
+import { WadRayMath }           from "aave-v3-core/contracts/protocol/libraries/math/WadRayMath.sol";
+import { IACLManager }          from "aave-v3-core/contracts/interfaces/IACLManager.sol";
+import { IPool }                from "aave-v3-core/contracts/interfaces/IPool.sol";
+import { IPoolConfigurator }    from "aave-v3-core/contracts/interfaces/IPoolConfigurator.sol";
+import { IScaledBalanceToken }  from "aave-v3-core/contracts/interfaces/IScaledBalanceToken.sol";
+
+import { CapAutomator } from "../src/CapAutomator.sol";
+
+contract CapAutomatorIntegrationTestsBase is Test {
+
+    address public constant POOL_ADDRESSES_PROVIDER = 0x02C3eA4e34C0cBd694D2adFa2c690EECbC1793eE;
+    address public constant POOL                    = 0xC13e21B648A5Ee794902342038FF3aDAB66BE987;
+    address public constant POOL_CONFIG             = 0x542DBa469bdE58FAeE189ffB60C6b49CE60E0738;
+    address public constant DATA_PROVIDER           = 0xFc21d6d146E6086B8359705C8b28512a983db0cb;
+    address public constant ACL_MANAGER             = 0xdA135Cd78A086025BcdC87B038a1C462032b510C;
+    address public constant SPARK_PROXY             = 0x3300f198988e4C9C63F75dF86De36421f06af8c4;
+    address public constant WETH                    = 0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2;
+    address public constant WBTC                    = 0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599;
+
+    address public user;
+
+    address[] assets;
+
+    CapAutomator public capAutomator;
+
+    IACLManager aclManager = IACLManager(ACL_MANAGER);
+    IPool pool             = IPool(POOL);
+
+    function setUp() public {
+        vm.createSelectFork(getChain("mainnet").rpcUrl, 18721430);
+
+        capAutomator = new CapAutomator(POOL_ADDRESSES_PROVIDER);
+
+        capAutomator.transferOwnership(SPARK_PROXY);
+
+        vm.prank(SPARK_PROXY);
+        aclManager.addRiskAdmin(address(capAutomator));
+
+        assets = pool.getReservesList();
+
+        user = makeAddr("user");
+    }
+
+}
+
+contract GeneralisedTests is CapAutomatorIntegrationTestsBase {
+
+    using ReserveConfiguration for DataTypes.ReserveConfigurationMap;
+    using WadRayMath           for uint256;
+
+    function test_E2E_increaseBorrow() public {
+        for(uint256 i; i < assets.length; i++) {
+            DataTypes.ReserveData memory reserveData = pool.getReserveData(assets[i]);
```

Comment:
I don't understand the comment

Reply:
Yeah it may be something with your issue creator @lucas-manuel . If you read these on Github they aren't clear what you are suggesting as a change. I have the same issue sometimes.

Reply:
Ya maybe no more code changes inside the IDE extension. It's completely unclear what you changed in Github:

<img width="777" alt="Screen Shot 2023-12-21 at 8 59 55 AM" src="https://github.com/marsfoundation/sparklend-cap-automator/assets/588921/5cc2e0f4-4dc2-4d5d-8ff4-b4f7b52dfcce">